1-js/13-modules/02-import-export/article.md

```diff
@@ -90,159 +89,159 @@ say.sayHi('John');
+Na maioria dos casos, a segunda abordagem é preferida, para que cada "coisa" tenha seu próprio módulo.
 
-Modules provide special `export default` syntax to make "one thing per module" way look better.
+Naturalmente isso exige muitos arquivos porque tudo precisa do seu próprio módulo, mas isso não é problema. Na verdade a navegação do código se torna mais fácil se os arquivos forem bem nomeados e estruturados em pastas.
 
-It requires following `export` and `import` statements:
+Módulos fornecem a sintaxe especial `export default` ("a exportação default") para melhorar a aparência de "uma coisa por módulo".
 
-1. Put `export default` before the "main export" of the module.
-2. Call `import` without curly braces.
-
-For instance, here `user.js` exports `class User`:
+Coloque `export default` antes da entidade a ser exportada:
 
 ```js
 // 📁 user.js
-export *!*default*/!* class User { // just add "default"
+export *!*default*/!* class User { // apenas adicione "default"
   constructor(name) {
     this.name = name;
   }
 }
 ```
 
-...And `main.js` imports it:
+Pode existir apenas um `export default` por arquivo.
+
+...E então importamos sem as chaves:
 
 ```js
 // 📁 main.js
-import *!*User*/!* from './user.js'; // not {User}, just User
+import *!*User*/!* from './user.js'; // não {User}, apenas User
 
 new User('John');
 ```
 
-Imports without curly braces look nicer. A common mistake when starting to use modules is to forget curly braces at all. So, remember, `import` needs curly braces for named imports and doesn't need them for the default one.
+Imports sem chaves parecem mais agradáveis. Um erro comum quando se começa a usar módulos é esquecer das chaves. Então, lembre-se, `import` precisa de chaves para import nomeados e não precisa para os imports default.
 
-| Named export | Default export |
+| Exporte nomeado | Export default |
 |--------------|----------------|
 | `export class User {...}` | `export default class User {...}` |
 | `import {User} from ...` | `import User from ...`|
 
-Naturally, there may be only one "default" export per file.
+Tecnicamente, podemos ter tanto exports nomeados como exports default em um único módulo, mas na prática as pessoas geralmente não misturam os dois. Um módulo possui exports nomeados ou o export default.
 
-We may have both default and named exports in a single module, but in practice people usually don't mix them. A module has either named exports or the default one.
+Como pode haver no máximo um export default por arquivo, a entidade exportada pode não ter nome.
 
-**Another thing to note is that named exports must (naturally) have a name, while `export default` may be anonymous.**
-
-For instance, these are all perfectly valid default exports:
+Por exemplo, todos esses default exports são perfeitamente válidos:
```

Comment:
swap default exports -> exports default ... makes it easier to read in "portuguese"